src/index.html

```diff
@@ -5,9 +5,41 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/styles.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <section class="card" data-qa="card">
+        <img
+          class="card__monitor"
+          src="./images/imac.jpeg"
+          alt="monitor"
+        >
+        <p class="card__name">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="reviews">
+          <div class="reviews__stars star">
+            <div class="reviews__stars--star star"></div>
```

Comment:
Fix classes everywhere
```suggestion
            <div class="stars__star"></div>
```